_locales/en/messages.json

```diff
@@ -953,6 +953,16 @@
     "description":
       "Confirmation dialog text that asks the user if they really wish to delete the conversation. Answer buttons use the strings 'ok' and 'cancel'. The deletion is permanent, i.e. it cannot be undone."
   },
+  "deleteContact": {
```

Comment:
Remove this if we don't use it?

Reply:
This is used as the text in the right click menu and in the cog